spring-amqp/src/main/java/org/springframework/amqp/core/Declarables.java

```diff
@@ -30,7 +34,14 @@
 
 	private final Collection<Declarable> declarables = new ArrayList<>();
 
+	public Declarables(Declarable... declarables) {
```

Comment:
I think we need to add here this:
```
@SafeVarargs
@SuppressWarnings("varargs")
```
See the same `Arrays.asList()`

Reply:
I don't see any warnings - I believe the Array case is because `T` is generic; here `Declarable` is a raw type.